packages/@aws-cdk/aws-codeartifact/test/integ.repository.ts

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+import * as cdk from '@aws-cdk/core';
+import * as codeartifact from '../lib';
+
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codeartifact');
+
+const domain = new codeartifact.Domain(stack, 'domain', { domainName: 'example-domain' });
```

Comment:
At least one integ test should be a complete example (e.g., policies, keys, repositories). One integ test with all of the above, plus an upstream and external repro, would be fine in place of these three.